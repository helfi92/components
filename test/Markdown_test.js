import React from 'react';
import Markdown from '../src/components/Markdown';

describe('<Markdown />', () => {
  it('should render correctly', () => {
    const text = '[Foo](Bar)';
    const markdown = mount(
      <Markdown>{text}</Markdown>
    );

    expect(markdown).toMatchSnapshot();
  });
});
