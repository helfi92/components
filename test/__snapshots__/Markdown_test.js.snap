// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Markdown /> should render correctly 1`] = `
<WithStyles(Markdown)>
  <Markdown
    classes={
      Object {
        "root": "Markdown-root-1",
      }
    }
  >
    <span
      className="Markdown-root-1"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<p><a href=\\"Bar\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Foo</a></p>
",
        }
      }
    />
  </Markdown>
</WithStyles(Markdown)>
`;
